aws-google-auth: add page

grem11n opened the pull request:
<!-- Thank you for sending a PR! -->
<!-- Please perform the following checks and mark all the boxes accordingly. -->
<!-- You can remove the checklist items that don't apply to your PR. -->

- Added a page for the `aws-google-auth` tool, which is useful if you're using AWS with SAML.

- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/tldr-pages/tldr?pullRequest=3287) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/tldr-pages/tldr?pullRequest=3287) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/tldr-pages/tldr?pullRequest=3287) it.</sub>

grem11n commented:
@andrik, thank you for your suggestion!
I have updated this PR: removed `awk` part and also updated the description.
The PR for `awk` doc changes are in this PR: https://github.com/tldr-pages/tldr/pull/3300

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+# aws-google-auth
+
+> A command-line tool allows you to acquire AWS temporary (STS) credentials using Google Apps as a federated (Single Sign-On, or SSO) provider.
```
```suggestion
> Command line tool to acquire AWS temporary (STS) credentials using Google Apps as a federated (Single Sign-On) provider.
```

Let's simplify this, shall we?

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+> A command-line tool allows you to acquire AWS temporary (STS) credentials using Google Apps as a federated (Single Sign-On, or SSO) provider.
+> More information: <https://github.com/cevoaustralia/aws-google-auth>.
+
+- Login with Google SSO:
```
```suggestion
- Login with Google SSO using the IDP and SP identifiers and set the credentials duration to one hour:
```
Let's expand here and explain what the arguments are.

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+
+- Login with Google SSO:
+
+`aws-google-auth -u example@example.com -I $GOOGLE_IDP_ID -S $GOOGLE_SP_ID -d 3600`
```
```suggestion
`aws-google-auth -u {{example@example.com}} -I {{$GOOGLE_IDP_ID}} -S {{$GOOGLE_SP_ID}} -d {{3600}}`
```
Wrap user tokens in curly braces.

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+
+`aws-google-auth -u example@example.com -I $GOOGLE_IDP_ID -S $GOOGLE_SP_ID -d 3600`
+
+- Ask, which role to use (in case you have several SAML roles):
```
```suggestion
- Login [a]sking which role to use (in case of several available SAML roles):
```
Little bit clearer.

sbrl replied:
I haven't seen the `[a]sking` syntax in other pages? Seems fine to me though.

mebeim replied:
I've seen it often enough:

	$ grep -r '\w\+\?\[.\]\w\+' pages

	pages/linux/xsel.md:- Use a command's output as input of the clip[b]oard (equivalent to `Ctrl + C`):
	pages/linux/pwgen.md:- Generate random password with s[y]mbols:
	pages/linux/isoinfo.md:- E[x]tract a specific file from an ISO image and send it out `stdout`:
	pages/common/perl.md:- Loo[p] over all lines of a file, editing them [i]n-place using a find/replace [e]xpression:
	pages/common/chmod.md:- Give the [u]ser who owns a file the right to e[x]ecute it:

sbrl replied:
Ah, cool. No need to worry then :P

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+
+- Ask, which role to use (in case you have several SAML roles):
+
+`aws-google-auth -u example@example.com -I $GOOGLE_IDP_ID -S $GOOGLE_SP_ID -d 3600 -a`
```
```suggestion
`aws-google-auth -u {{example@example.com}} -I {{$GOOGLE_IDP_ID}} -S {{$GOOGLE_SP_ID}} -d {{3600}} -a`
```
Wrap user tokens in curly braces.

mebeim commented on pages/common/aws-google-auth.md:
```diff
@@ -0,0 +1,20 @@
+
+- Resolve aliases for AWS accounts:
+
+`aws-google-auth -u example@example.com -I $GOOGLE_IDP_ID -S $GOOGLE_SP_ID -d 3600 -a --resolve-aliases`
```
```suggestion
`aws-google-auth -u {{example@example.com}} -I {{$GOOGLE_IDP_ID}} -S {{$GOOGLE_SP_ID}} -d {{3600}} -a --resolve-aliases`
```
Wrap user tokens in curly braces.

grem11n commented:
@mebeim, thank you for suggestions!
I have updated my PR
